### PR TITLE
Added rails 7

### PR DESCRIPTION
**Pull request by IsmaelMarinMolina**

### CODE REVIEW

**Comment by dom111** [comment]
This looks good! I'd suggest updating the README to include notes about how it's been technically tested on Rails 7, but if integrating with Rails 7 and you encounter any issues, please contact us in #customerdatateam on Slack. What do you think?

**Review by dom111 — APPROVED** [review]

File: .drone.yml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    DOCKER_USERNAME:
+      from_secret: docker_username
+    DOCKER_PASSWORD:
+      from_secret: docker_password
+  jfrog_credentials: &jfrog_credentials
+    BUNDLE_FUNDINGCIRCLE__JFROG__IO:
+      from_secret: bundle_fundingcircle__jfrog__io
+  test_commands: &test_commands
+    commands:
+      - wget -O vault.zip -q https://releases.hashicorp.com/vault/${VAULT_VERSION}/vault_${VAULT_VERSION}_linux_amd64.zip
```
Comment (andrey-fc) [inline .drone.yml:30]: where do you get VAULT_VERSION from?

File: .drone.yml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+      - bundle exec rake app:db:create
+      - bundle exec rake app:db:schema:load
+      - bundle exec rake app:db:test:prepare
+      - bundle exec rake
+
+concurrency:
+  limit: 1
+
+steps:
+  - name: build_ruby4
```
[the remark this answers is not in the document]
Comment (IsmaelMarinMolina) [reply .drone.yml]: Hellooooo, 

So not sure, already mentioned to @dom111  in the other PR. Don't know which applications still using rails 4 so feel unsure to remove it. Anyway, I can remove the rails 4 from here if you agree to do it 👍🏼

File: .drone.yml (hunk 1 above)
Comment (IsmaelMarinMolina) [reply .drone.yml]: it is a secret in the repository, to be honest. I don't know where the previous one was defined looked for it in the vault and in the seed but any of them contains that variable 🤷🏼

**Pull request merged**